Sources/Diagnose/CommandLineArgumentsReducer.swift

```diff
@@ -12,8 +12,19 @@
 
 import Foundation
 
+// MARK: - Entry point
+
+extension RequestInfo {
+  func reduceCommandLineArguments(using executor: SourceKitRequestExecutor) async throws -> RequestInfo {
```

Comment:
Occurs to me this could also be on the executor now. Up to you which you prefer.

Reply:
Let’s keep it on ` RequestInfo` to match `reduceInputFile`